modules/qr_code.py

```diff
@@ -0,0 +1,59 @@
+import qrcode
+import uuid
+import os
+import logging
+
+from .args import get_args
+
+args = get_args()
+qr = qrcode.QRCode()
+logger = logging.getLogger(__name__)
+
+QRCODE_FOLDER = args.qr_code_folder_path 
+MAX_CACHE_SIZE = args.qr_code_cache_size
```

Comment:
i think we can remove `args` and `qr`, then pass the folder and cache size to the constructor from server.py

Reply:
Resolved from commit [53c15d0](https://github.com/SCE-Development/cleezy/pull/39/commits/53c15d052dc2d15708838fa9b2e46187991acc0e) and [b56884c](https://github.com/SCE-Development/cleezy/pull/39/commits/b56884cb36eb3832aefc14a750431d5ae1d12134)